htsohm/htsohm.py

```diff
@@ -262,7 +262,7 @@ def get_all_parent_ids(run_id, generation):
     result.close()
     return parent_ids
 
-def get_mutation_strength(run_id, generation, parent):
+def calculate_mutation_strength(run_id, generation, parent):
```

Comment:
Move calculate_percent_children_in_bin off of material and just make it a function of bin in htsohm. Then calculate_mutation_strength won't need the parent either and can just be a function of bin.